File-Storage-Security/Deployment/aws-python-deploy-to-all-existing-s3/deploy.py

```diff
@@ -220,6 +212,7 @@ def add_to_cloudone(ws_api, stack_id, storage_stack):
         body=encoded_msg,
     )
     transform = json.loads(resp.data.decode("utf-8"))
-    url = "https://filestorage."+cloud_one_region+".cloudone.trendmicro.com/api/stacks/"+transform['stackID']
+    url = "https://filestorage." + cloud_one_region + ".cloudone.trendmicro.com/api/stacks/" + transform['stackID']
```

Comment:
Same here, no spaces pls

Reply:
@felipecosta09  Spaces are okay. They are not in the url itself here